samples/main.cpp

```diff
@@ -210,6 +210,42 @@ void TestServo() {
     brain.sleep(200);
 }
 
+void TestAccelerometer() {
+    int xAxis = brain.accelerometer.getX();
+    int yAxis = brain.accelerometer.getY();
+    int zAxis = brain.accelerometer.getZ();
+
+    if (brain.accelerometer.getGesture() == ACCELEROMETER_EVT_FACE_DOWN)
```

Comment:
Use a switch, not an if. Set the gesture string in the switch cases, don't call DrawString itself.

```cs
const char* msg = nullptr;
switch (brain.acceleroemter.getGesture()) {
    case ACCELEROMETER_EVT_TILT_RIGHT: msg = "TILT_RIGHT"; break;
    ...
}

if (msg != nullptr)
    DrawString(1, 36, msg);
```